skyportal/handlers/api/photometry.py

```diff
@@ -181,6 +183,18 @@ def post(self):
             for key in packet:
                 packet[key] = nan_to_none(packet[key])
 
+            # Check if alert ID already exists & only update groups if so
+            alert_id = hash((packet["obj_id"], packet["mjd"]))
```

Comment:
Sometimes this can return a negative integer, are we cool with that? 

Example,

```python
# tuple of vowels
vowels = ('a', 'e', 'i', 'o', 'u')
print('The hash is:', hash(vowels))
# The hash is: -695778075465126279
```

Reply:
I don't see why notd